packages/kernel-modules/kernel-modules/finalize.yaml

```
@@ -1,2 +1,2 @@
 install:
-- depmod -a
\ No newline at end of file
+- depmod -a $(ls /lib/modules)
```

What if there is more than one? I'd suggest to go with head:
```suggestion
- depmod -a $(ls /lib/modules | head -n1)
```

---

or we could do depmod of all of them. We really should consider to create a helper here

---

We can run depmod for all kernel with a `for` loop at the moment.

Indeed, we only need to run depmod for the kernel for which the kernel module is installed. 
At the moment, mOS provide only two kernel (lts and no-lts). Maybe a combo  `index .Values.lts ` and `luet search --installed ` to get the kernel version associated to the current install. 
I think the better is to rely on something like `eselect` in the long term.
How kernel switching is planned in mOS?

---

updated